src/main/java/org/jabref/gui/fieldeditors/JournalEditorViewModel.java

```diff
@@ -22,8 +22,11 @@ public void toggleAbbreviation() {
             return;
         }
 
-        if (journalAbbreviationRepository.isKnownName(text.get())) {
-            Optional<String> nextAbbreviation = journalAbbreviationRepository.getNextAbbreviation(text.get());
+        // Ignore brackets when matching abbreviations.
+        final String name = text.get().replaceAll("[{}]", "");
+
```

Comment:
You can use the existing method StringUtil.ignoreCurlyBracket(text)

Reply:
Ah, thank you. Updated the code to use that method instead.